app/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser.rb

```diff
@@ -318,7 +318,8 @@ def parse_config_pattern(config_pattern)
     def get_host_relationship(node)
       # Assign a physicalserver and host if server already exists and
       # some host match with physical Server's serial number
-      Host.find_by(:service_tag => node.serialNumber)
+      Host.find_by(:service_tag => node.serialNumber) ||
+        Host.joins(:hardware).find_by('hardwares.serial_number' => node.serialNumber)
```

Comment:
It looks like there is an inconsistency with the different parsers, vmware sets service_tag but ovirt and scvmm do not.

Reply:
@rodneyhbrown7 I updated the SCVMM provider to put the `PhysicalMachine/SMBiosGUID` in `hardware.serial_number` and @jameswnl is in the process of adding the VMware Host's UUID to the same place.

Reply:
@rodneyhbrown7 okay VMware has been updated https://github.com/ManageIQ/manageiq-providers-vmware/pull/139 so every infra provider should put the bios uuid in `hardware.serial_number`

Can you do a `where` instead of a find_by?  It is possible that there are duplicates since this comes from host firmware

Reply:
@rodneyhbrown7 could you change `hardwares.serial_number` to `hardware.serial_number` like @agrare said? And use `where` instead of `find_by`.

Reply:
I am testing this and having problems successfully making the association.   I will continue to evaluate the association here.

Reply:
I still think we have an issue with linking to the wrong host if there are duplicate serial numbers

Reply:
I am not opposed to resolving that issue, I just think that it would be unexpected to see the BIOS id in the serial number field.

Reply:
Removing WIP.   We will work on a second PR to that will look for uuid or serial number in the hardwares.serial_number column.  This PR will enable RHEVM hosts to be discovered.

@miq-bot remove_label wip